package.json

```diff
@@ -15,11 +17,22 @@
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
+    "@leocode/eslint-config": "1.1.0",
     "@nestjs/common": "^7.6.13",
     "@nestjs/core": "^7.6.13",
+    "@nestjs/platform-express": "8.0.6",
+    "@nestjs/testing": "7.6.13",
     "@rollup/plugin-typescript": "^8.2.0",
+    "@types/jest": "26.0.24",
+    "@typescript-eslint/eslint-plugin": "4.29.0",
+    "@typescript-eslint/parser": "4.29.0",
+    "eslint": "7.32.0",
+    "eslint-plugin-import": "2.23.4",
+    "jest": "27.0.6",
     "reflect-metadata": "^0.1.13",
     "rollup": "^2.40.0",
+    "rxjs": "^7.3.0",
```

Comment:
`rxjs` `@nestjs/platform-express` and `@nestjs/testing` are needed to run nest application in e2e tests